influxql/iterator.gen.go.tmpl

```diff
@@ -1271,3 +1273,9 @@ func (enc *IteratorEncoder) encodeStats(stats IteratorStats) error {
 }
 
 {{end}}
+
```

Comment:
I was debating that. I was trying to keep it in the same file but I don't have a strong opinion on it. I can move it.